sbin/submitty_daemon_jobs/submitty_jobs/jobs.py

```diff
@@ -317,6 +318,85 @@ def run_job(self):
         os.chdir(current_path)
 
 
+class UpdateDockerData(AbstractJob):
+    '''
+    Performs the commands 'docker info' & 'docker images' and saves the 
+    results to a json file where the PHP user can read the data, used for
+    the submitty docker interface
+    '''
+    def format_bytes(size):
+        power = 2**10
+        n = 0
+        power_labels = {0 : '', 1: 'KB', 2: 'MB', 3: 'GB', 4: 'TB'}
+        while size > power:
+            size /= power
+            n += 1
+        
+        ret = str(math.floor(size))
+        if n in power_labels:
+            ret += power_labels[n]
+
+        return ret
+
+
+    def run_job(self):
+
+        tgt_path = Path(DATA_DIR, 'docker_data', 'submitty_docker.json')
+        log_path = Path(DATA_DIR, 'logs', 'docker_interface_logs')
+
+        today = datetime.datetime.now()
+        log_file_path = os.path.join(log_path, 
+                                "{:04d}{:02d}{:02d}.txt".format(today.year, today.month,
+                                today.day))
+        full_time = today.strftime("%m/%d/%Y %H:%M:%S")
+
+        docker_client = docker.from_env()
+        try:
+            docker_info = docker_client.info()
+            dokcer_client = docker.APIClient(base_url='unix://var/run/docker.sock')
+            images = docker_client.images()
+
+            #format images closer to output of 'docker images'
+            docker_info = json.loads(docker_info)
+            docker_images = json.loads(docker_images)
+            
+            for x in docker_images:
+                created_date = datetime.datetime.fromtimestamp(x['Created'])
+
+                diff = today - created_date
+                print(diff.days, math.floor( diff.days/30))
+
+                if (diff.days/ (30*12) > 0):
+                    x['CreatedSince'] = str(math.floor(diff.days/(30*12))) + " years ago"
+                elif (diff.days/30 > 0):
+                    x['CreatedSince'] = str(math.floor(diff.days/30)) + " months ago"
+                else:
+                    x['CreatedSince'] = str(diff.days) + " days ago"
```

Comment:
This type of stuff should be calculated on the front-end display, not here, especially since as soon as the date changes, this field will not be correct.